mod/01_rating/set/right/metric_value_filter.rb

```diff
@@ -0,0 +1,142 @@
+def virtual?
+  true
+end
+
+def industry_metric_name
+  'Richard Mills+Sector Industry'
+end
+
+def industry_value_year
+  '2015'
+end
+
+format :html do
+  def page_link text, page, _current, options={}
+    @paging_path_args[:offset] = page * @paging_limit
+    filter_args = {}
+    [:sort, :cited, :claimed, :company, :topic, :tag].each do |key|
+      filter_args[key] = params[key] if params[key].present?
+    end
+    options[:class] = 'card-paging-link slotter'
+    options[:remote] = true
+    link_to raw(text), path(@paging_path_args.merge(filter_args)), options
+  end
+
+  view :no_search_results do |_args|
+    %(
+      <div class="search-no-results">
+        No result
+      </div>
+    )
+  end
+
+  def filter_categories
+    %w(company industry project)
+  end
+
+  def filter_active?
+    Env.params.keys.any? { |key| filter_categories.include? key }
+  end
+
+  def default_core_args args
+    args[:buttons] = [
+      card_link(card.left, path_opts: { view: :content_left_col },
+                           text: 'Reset', class: 'slotter btn btn-default',
+                           remote: true),
+      button_tag('Filter', situation: 'primary', disable_with: 'Filtering')
```

Comment:
"Filter" should probably  be ajax, too.  Feels like a little lost context when loading takes you back to the top of the page.